### PR TITLE
Update issue-tracking.rst with links etc

### PR DESCRIPTION
**Pull request by ktoso**
cc @patriknw

### CODE REVIEW

File: akka-docs/rst/project/issue-tracking.rst [hunk 1]
```diff
@@ -35,4 +31,14 @@ have registered a GitHub user account.
 
 Thanks a lot for reporting bugs and suggesting features!
 
+Submitting Pull Requests
+------------------------
+
+.. note:: *A pull request is worth a thousand +1's.* -- Old Klangian Proverb
+
+Pull Requests fixing issues or adding functionality are are very welcome.
```
Comment (patriknw) [inline akka-docs/rst/project/issue-tracking.rst]: are are

**Comment by patriknw** [comment]
LGTM after typo

File: akka-docs/rst/project/issue-tracking.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/project/issue-tracking.rst]: whoops fixed

**Comment by ktoso** [comment]
Thanks for spotting that one

**Pull request merged**